tests/trace/test_saved_view.py

```diff
@@ -0,0 +1,172 @@
+import datetime
```

Comment:
Can you add some tests around the publishing / consuming of saved views - essentially testing the actual create/edit flow that you will need to hook up to the UI

Reply:
Added more tests including of loading back a published view.